config/pipelines/high_throughput_cardinal.wip.yml

```diff
@@ -23,7 +24,7 @@ Cardinal:
 Cardinal Banking:
   filters:
     request_type_key: limber_cardinal_banking
-    library_type: Cardinal Banking
```

Comment:
It seems to me like this library_type here should match whatever we create as part of the submission. You've removed it as a requirement for going down the pipeline here, but haven't removed it in config/purposes/cardinal.wip.yml on line 128